emgapi/views.py

```diff
@@ -1165,6 +1161,57 @@ def list(self, request, *args, **kwargs):
         return super(PublicationViewSet, self).list(request, *args, **kwargs)
 
 
+class GenomeCatalogueViewSet(mixins.RetrieveModelMixin,
+                             emg_mixins.ListModelMixin,
+                             emg_viewsets.BaseGenomeCatalogueGenericViewSet):
+
+    filter_class = emg_filters.GenomeCatalogueFilter
+
+    lookup_field = 'catalogue_id'
+    lookup_value_regex = '[^/]+'
+
+    queryset = emg_models.GenomeCatalogue.objects.all()
+
+    def retrieve(self, request, *args, **kwargs):
```

Comment:
Why are you overwriting these methods to only call the `super` implementation? ISn't this something that would happen just by inheritance. Is it for the documentation? Or Am I missing something here?

Reply:
It is purely to add the doc strings which then appear in the swagger documentation – this is the pattern used currently in the codebase to achieve this, although there are other patterns we could adopt globally instead...

Reply:
Cool. Happy with that pattern, I was just curious.